contracts/Account.cairo

```diff
@@ -194,6 +194,7 @@ func execute{
         selector: felt,
         calldata_len: felt,
         calldata: felt*,
+        nonce: felt
     ) -> (response : felt):
     alloc_locals
```

Comment:
That's not necessary since we're only allowing for sequential nonces, while the interface supports custom nonce mechanisms. I think just ignoring the parameter results in less anf more efficient code.

Reply:
_anf_

![image](https://user-images.githubusercontent.com/1703831/140976429-3a180b7c-a21d-4617-91ef-30e90da3e1fb.png)